### PR TITLE
feat: add PTR as an alias for I64

### LINKED ISSUE #15: Add `PTR` alias for `I64` ValType
This should be used to signify an Extism pointer instead of a 64-bit integer value. See https://github.com/extism/extism/pull/558

### PR DESCRIPTION
**Pull request by mhmd-azeez**
While the new test case works, I am not 100% sure if this is all what we need. Another pair of eyes that has more experience with Java would be appreciated

Fixes #15

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bhelx) [inline .idea/compiler.xml:1]: can we add this folder to .gitignore if it isn't already? I think this is IDE specific stuff.

Comment (mhmd-azeez) [reply .idea/compiler.xml]: removed the folder and added it to gitignore

**Comment by bhelx** [comment]
Sorry if i'm a bit out of the loop on this. Is this change necessary? I think we did this in rust so the convert crate could disambiguate b/w the user wanting to pass an i64 vs a ptr but i'm not sure that distinction exists here in the Java library.

**Comment by mhmd-azeez** [comment]
I actually haven't spoken to @zshipko about this, but my assumption was:
1. It let's people make intentions explicit (i.e. document that they want/return a pointer to a memory, not just a normal i64)
2. It keeps things consistent with the other SDKs

But he probably has more insight about this

**Comment by zshipko** [comment]
I do think this makes things a bit more clear for users who don't know about wasm native types. We can document that I64 should always be used, but I think having PTR makes things a little less confusing.

**Review by bhelx — APPROVED** [review]
Okay I'm cool with this!

**Pull request merged**